README.md

```diff
@@ -99,26 +99,53 @@ cargo near build
       template project, generated by `cargo near new`
 
-Important flags:
-
-1. `--no-docker`
-    - flag can be used to perform a regular build with rust toolchain installed onto host, running the `cargo-near` cli. 
-    - *NO*-Docker builds run against actual state of code in filesystem and not against a version, committed to source control.   
+**What's a reproducible build in context of NEAR?**
+Why is it needed? Explanation of these points and a step-by-step tutorial is present at [SourceScan/verification-guide](https://github.com/SourceScan/verification-guide).
 
-2. `--no-locked` 
-    - flag is allowed in *NO*-Docker builds, e.g. to generate a `Cargo.lock` *and* simultaneously build the contract.
-    - flag is allowed in Docker builds, but 
-      - such builds are not reproducible due to potential update of dependencies and compiled `wasm` mismatch as the result.
+<details>
+  <summary>Additional (optional) details on possible <code>[package.metadata.near.reproducible_build]</code> configuration</summary><p>
+  
+1. available images can be found by this link https://hub.docker.com/r/sourcescan/cargo-near/tags 
+  - [`image`](https://github.com/near/cargo-near/blob/main/cargo-near/src/commands/new/new-project-template/Cargo.template.toml#L18) and [`image_digest`](https://github.com/near/cargo-near/blob/main/cargo-near/src/commands/new/new-project-template/Cargo.template.toml#L19) are straightforward to configure:
+  ![image_and_digest_pinpoint](./docs/image_and_digest_pinpoint.png)
+2. build command flags can be configured, if needed, by changing [`container_build_command`](https://github.com/near/cargo-near/blob/main/cargo-near/src/commands/new/new-project-template/Cargo.template.toml#L29) field
+  - base `container_build_command` for images prior to **sourcescan/cargo-near:0.13.0-rust-1.83.0** is `["cargo", "near", "build"]` 
+  - base `container_build_command` for images starting with **sourcescan/cargo-near:0.13.0-rust-1.83.0** and after it  is `["cargo", "near", "build", "non-reproducible-wasm", "--locked"]`, where the `--locked` flag is required
+  - additional flags, if needed, can be looked up on
+    - `cargo near build non-reproducible-wasm --help` for newer/latest images 
+    - `cargo near build --help` for older ones 
+    - running `docker run -it sourcescan/cargo-near:0.11.0-rust-1.82.0` (or another specific image) and checking the `--help` message of exact `cargo-near` in container may be helpful when in doubt    
+3. `cargo near` allows parameterizing build with values of environment variables, present at the time of the build and not present in a contract's source code,
+   by specifying their names in [`passed_env`](https://github.com/near/cargo-near/blob/main/cargo-near/src/commands/new/new-project-template/Cargo.template.toml#L24) array
+  - supported by `sourcescan/cargo-near:0.10.1-rust-1.82.0` image or later images
+  - SourceScan/Nearblocks does not support verifying such contracts yet. 
+
+</p></details>
```

Comment:
this is a collapsable section 
![image](https://github.com/user-attachments/assets/60953d66-46f5-43f5-a1ad-84f0c88f280d)